src/components/inspectors/InspectorPanel.vue

```diff
@@ -173,6 +173,7 @@ export default {
     defaultInspectorHandler(value) {
       /* Go through each property and rebind it to our data */
       for (const key in omit(value, ['$type', 'eventDefinitions'])) {
+        this.updateState();
```

Comment:
@chipit24  While digging into the code saw that the 'save-state' emits when focusing out of the inspector.  Using that same method, I added it to this function whenever the form renderer is updated. Not sure if this is the best place to input this method, but that is the idea I had behind this.

Reply:
This is a great catch! But, I'm hesitant to have this merged without clarifying a few things:

Updating on every keystroke saves a new BPMN string to the undo/redo stack, so when you press "undo" after entering a name, it will undo the name you entered letter-by-letter. This was a choice we consciously made, as we wanted updates like this to be batched. Has this requirement changed? If not, we'll have to come up with a different solution to this problem. Maybe only make the additional call to `updateState` when the validation status of the form changes?

Also, where you have the `updateState` function call right now means it will only be executed for the default case, but it should be called for custom inspector handlers as well.

Reply:
The requirements have not changed, so another solution is needed. I'll look into your suggestion for adding the `updateState` when the validation status of the form changes. Thank you!